client/src/components/Header/SearchBar/TextSearch/TextSearch.js

```diff
@@ -31,16 +31,15 @@ export default class TextSearch extends React.Component {
 
     render() {
         const { searchPlaceholder, searchQuery } = this.state;
-        const { className, onKeyDown, width, barBottomSpacing } = this.props;
+        const { className, onKeyDown } = this.props;
         return (
-            <div className="position-relative" style={{width: width, marginBottom: barBottomSpacing}}>
+            <div className="text-search position-relative">
```

Comment:
Since I needed to target this element from `Header.scss` I just added a class name to this and did the same for other cases such as `location-search` and `search-button`